packages/cli/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "@vue-storefront/cli",
+  "version": "0.0.4",
```

shouldn't we keep versioning compatible for easier maintaining?
```suggestion
  "version": "1.10.0-rc.1",
```